app/src/validator/user.js

```diff
@@ -0,0 +1,36 @@
+const { validate, Joi } = require('express-validation');
+
+const schema = {
+  searchUsers: {
+    query: Joi.object({
+      userId: Joi.array().items(Joi.string().guid({
+        version: 'uuidv4'
+      })),
+      identityId: Joi.array().items(Joi.string().guid({
+        version: 'uuidv4'
+      })),
+      idp: Joi.array().items(Joi.string()),
+      username: Joi.string(),
+      email: Joi.string().email(),
+      firstName: Joi.string(),
+      fullName: Joi.string(),
+      lastName: Joi.string(),
+      active: Joi.boolean(),
```

Comment:
Jeremy's isTruthy method means any of these could be valid: `'true', 't', 'yes', 'y', '1']`